tests/VerificationCodeTest.php

```diff
@@ -173,4 +175,42 @@ public function it_returns_false_if_test_verifiable_with_empty_test_code()
 
         $this->assertFalse(VerificationCodeFacade::verify('', $verifiable));
     }
+
+    /** @test */
+    public function it_works_if_notification_config_value_is_empty()
+    {
+        $email = $this->faker->safeEmail;
+
+        VerificationCodeFacade::send($email);
+
+        $this->assertNotNull(VerificationCode::where('verifiable', $email)->first());
+
+        Notification::assertSentTo(
+            new AnonymousNotifiable,
+            VerificationCodeCreated::class,
+            function ($notification, $channels, $notifiable) use ($email) {
+                return $notifiable->routes['mail'] === $email;
+            });
+    }
+
+    /** @test */
+    public function it_throws_an_error_if_notification_does_not_extend_the_verification_notification_class()
+    {
+        $this->expectException(InvalidClassException::class);
+        $this->expectExceptionMessage('The notification should extend the VerificationCodeInterface.');
+        $email = $this->faker->safeEmail;
+
+        config()->set('verification-code.notification', WrongNotification::class);
+
+        VerificationCodeFacade::send($email);
+
+        $this->assertNotNull(VerificationCode::where('verifiable', $email)->first());
+
+        Notification::assertSentTo(
+            new AnonymousNotifiable,
+            VerificationCodeCreated::class,
+            function ($notification, $channels, $notifiable) use ($email) {
+                return $notifiable->routes['mail'] === $email;
+            });
```

Comment:
this does not run I think, because no notification is being sent if the notification-class is incorrect. So this code can be removed probably?